docs/content/preview/explore/query-1-performance/pg-stat-progress-create-index.md

```diff
@@ -0,0 +1,62 @@
+---
+title: View CREATE INDEX status with pg_stat_progress_create_index
+linkTitle: View CREATE INDEX status
+description: Use pg_stat_progress_create_index to get the CREATE INDEX command status, including the status of an ongoing concurrent index backfill, and the index build's progress reports.
+headerTitle: View CREATE INDEX status with pg_stat_progress_create_index
+image: /images/section_icons/index/develop.png
+menu:
+  preview:
+    identifier: pg-stat-progress-create-index
+    parent: query-tuning
+    weight: 450
+type: docs
+---
+
+YugabyteDB can build indexes on non-empty tables online, without failing other concurrent writes. When you add a new index to a table that is already populated with data, you can use the YSQL [`CREATE INDEX`](../../../api/ysql/the-sql-language/statements/ddl_create_index/#semantics) statement to enable building these indexes in an online manner, without requiring downtime. For details how online backfill of indexes works, see the [Online Index Backfill](https://github.com/yugabyte/yugabyte-db/blob/master/architecture/design/online-index-backfill.md) design document.
```

Comment:
The view shows entries for both non-concurrent and concurrent index builds - I think based on this introduction (which only talks about concurrent index builds) that might not be clear?